packaging/rpm/cvmfs-universal.spec

```diff
@@ -771,6 +788,8 @@ systemctl daemon-reload
 
 %changelog
 # - When using fuse3, require at least version 3.3.0 (for premounting).
+* Thu Jul 31 2025 Valentin Volkl <vavolkl@cern.ch>> - 2.14.0
```

Comment:
So if this isn't until 2.14.0, for 2.13.3 are you going to directly change what's now a patch in 2.13.2-2?  I don't think it's good to carry the patch forward.

Reply:
There will need to be a 2.13.3 at least for #3947